CHANGELOG.md

```diff
@@ -16,6 +19,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Accessibility - Announce enlargement of captured image in preview
 - UI: Accessibility - Announce camera alerts
 - UI: Accessibility - Announce validation errors and warnings on confirm screen
+- UI: Accessibility - Update all visually obvious lists to use the relevant HTML list elements
```

Comment:
How come it's within a previous release?

Reply:
Yeap. This definitely should not be in here. CHANGELOG for 5.2.0 release has it under the right release. Will remove from here.